packages/oas-extensions/package.json

```diff
@@ -39,7 +39,7 @@
     "prepack": "npm run build",
     "test": "vitest run --coverage"
   },
-  "peerDependencies": {
```

Comment:
This was originally a peer dep in https://github.com/readmeio/oas-extensions to save me the hassle of bumping it whenever I published a major release of `oas` but because they're all in the same repository now it makes sense to just have this listed as a regular dependency.